### PR TITLE
Deprecating a bunch of useless functions.

### PR DESCRIPTION
**Pull request by moufmouf**
See #15

### CODE REVIEW

**Comment by stof** [comment]
does Mouf support reporting deprecations using the convention introduced in symfony (i.e. `@trigger_error('...', E_USER_DEPRECATED)`, so that error handlers not supporting the convention don't convert the deprecation warning to an exception thus breaking things) ?
If no, you should probably provide an integration of the `Twig_Util_DeprecationCollector`, to find usage of deprecated Twig features

File: src/Mouf/Html/Renderer/Twig/MoufTwigExtension.php [hunk 1]
```diff
@@ -116,6 +128,12 @@ public function translate(array $args = array())
         return call_user_func_array('iMsgNoEdit', $args);
     }
 
+    public function translateFilter($text, array $args = array())
```
Comment (stof) [inline src/Mouf/Html/Renderer/Twig/MoufTwigExtension.php]: given that the `translate` function would still require at least 1 argument, you would make this change directly in `translate` and use the same implementation for the function and the filter

**Comment by stof** [comment]
should the `t` function be deprecated in favor of the `t` filter ?

File: src/Mouf/Html/Renderer/Twig/MoufTwigExtension.php (hunk 1 above)
Comment (moufmouf) [inline src/Mouf/Html/Renderer/Twig/MoufTwigExtension.php]: Oh nice! :+1:

**Comment by moufmouf** [comment]
> should the t function be deprecated in favor of the t filter ?

True.

**Comment by moufmouf** [comment]
> If no, you should probably provide an integration of the Twig_Util_DeprecationCollector, to find usage of deprecated Twig features

I'll open a separate issue for this.

**Pull request merged**